### DISCUSSION TITLE
two profile menu

### DISCUSSION THREAD

**Discussion by whudn** [colinwilson/lotusdocs#107] [Feature Requests]
Can you create a view with two different profiles such as Docs Menu, and Blog Menu with their own styles.

**Comment by colinwilson** [comment]
Sorry, I'm not sure exactly what you mean. Could you elaborate?

**Comment by whudn** [comment]
I mean can you create a version for the blog, it's so amazing amazing

**Comment by colinwilson** [comment]
Sorry. The theme's focus is just documentation (for now). Maybe at some point in the future.
